tests/e2e/steps/steps_def.go

```diff
@@ -811,40 +934,42 @@ func configureAuthorizationProxyServer(cr csmv1.ContainerStorageModule) error {
 		"--array-insecure",
 		"--insecure", "--addr", fmt.Sprintf("csm-authorization.com:%s", port),
 	)
-	fmt.Println("=== Storage === ", cmd.String())
+	fmt.Println("=== Storage === \n", cmd.String())
```

Comment:
I don't see this print in the logs in the description. Does this print the actual `\n` character or does it print a new line? `fmt.Println` already prints a new line.

Reply:
I removed the configuration output since internal resources are referenced. It add a  new lines, it does not print the character.
```
 === Configuring Authorization Proxy Server ===
=== Generating Admin Token ===
=== Copying Admin Token to Tmp File ===
 
=== Creating Storage ===
 
=== Storage === 
karavictl ...
=== Creating Tenant ===
```